bo/angebot.go

```diff
@@ -14,7 +14,7 @@ import (
 // die jeweils für eine oder mehrere Marktlokationen erstellt werden.
 type Angebot struct {
 	Geschaeftsobjekt
-	Angebotsnummer              string                 `json:"angebotsnummer,omitempty" validate:"required"` // Eindeutige Nummer des Angebotes.
+	Angebotsnummer              *string                `json:"angebotsnummer,omitempty" validate:"required"` // Eindeutige Nummer des Angebotes.
```

Comment:
Hat das required noch eine Daseinsberechtigung, wenn alles optional ist?
```suggestion
	Angebotsnummer              *string                `json:"angebotsnummer,omitempty"` // Eindeutige Nummer des Angebotes.
```